src/Components/CreateImageWizard/steps/ImageOutput/ReleaseSelect.tsx

```diff
@@ -75,6 +82,11 @@ const ReleaseSelect = () => {
     );
 
     filteredRhel.forEach((value, key) => {
+      // Only show RHEL 9 beta in preview
```

Comment:
Is this a permanent thing, or do we want to move it to stable later?

Reply:
i'm not sure! I'm happy to have it instable too